modules/apps/object/object-api/src/main/java/com/liferay/object/constants/ObjectWebKeys.java

```diff
@@ -43,6 +43,10 @@ public class ObjectWebKeys {
 
 	public static final String OBJECT_LAYOUT = "OBJECT_LAYOUT";
 
+	public static final String
+		OBJECT_PORTLET_DEFINITION_DETAILS_DISPLAY_CONTEXT =
+			"OBJECT_PORTLET_DEFINITION_DETAILS_DISPLAY_CONTEXT";
```

Comment:
@igor-franca please rename both  to `OBJECT_DEFINITIONS_DETAILS_DISPLAY_CONTEXT` (same name as the `ObjectDefinitionsDetailsDisplayContext` class). Thanks!

Reply:
Done 👍🏾